### ISSUE TITLE
The default log level should include errors

### ISSUE THREAD

**Issue by paulvonallwoerden** [hasura/graphql-engine#5592]
### Summary

Currently logs of level "error" are not written by default. This is counter intuitive as I would expect that the default log level would include errors.

Example. I tried to start graphql-engine with:

```bash
graphql-engine serve
```

The server now just prints

```json
{"type":"startup","timestamp":"2020-08-14T14:49:17.283+0000","level":"info","detail":{"kind":"server_configuration","info":{"live_query_options":{"batch_size":100,"refetch_delay":1},"transaction_isolation":"ISOLATION LEVEL READ COMMITTED","plan_cache_options":{"plan_cache_size":null},"enabled_log_types":["http-log","websocket-log","startup","webhook-log"],"server_host":"HostAny","enable_allowlist":false,"log_level":"info","auth_hook_mode":null,"use_prepared_statements":true,"unauth_role":null,"stringify_numeric_types":false,"enabled_apis":["metadata","graphql","config","pgdump"],"enable_telemetry":true,"enable_console":false,"auth_hook":null,"jwt_secret":null,"cors_config":{"allowed_origins":"*","disabled":false,"ws_read_cookie":null},"console_assets_dir":null,"admin_secret_set":false,"port":8080}}}
```

and exits. It took me a long time to figure out that errors are just not included in the default log level.
It would be prefered if the error would have been written like if I had run:

```bash
graphql-engine serve --log-level error
```

which would have shown me the reason.

```json
{"type":"startup","timestamp":"2020-08-14T14:50:41.876+0000","level":"error","detail":{"kind":"db_migrate","info":{"path":"$","error":"Cannot use database previously used with a newer version of graphql-engine (expected a catalog version <=34, but the current version is 36).","code":"not-supported"}}}
```

### System as it is currently

Errors are not included in the logs by default, which makes it hard to find the cause of failure.

### System as I would expect it

Errors are included in the default logs.

### Info

This has been done with the Docker Image `hasura/graphql-engine:v1.2.1.cli-migrations-v2`

**Comment by tirumaraiselvan** [comment]
@paulvonallwoerden This was a regression in v1.2.1 which was fixed in v1.2.2 via https://github.com/hasura/graphql-engine/pull/4801

If you upgrade, you would see the error log printed by default.

**Comment by paulvonallwoerden** [comment]
@tirumaraiselvan Thank you for the clarification.
